### ISSUE TITLE
[Bug] Utf8 doesn't work in 4.5.0.3

### ISSUE THREAD

**Issue by AlexisHuvier** [ChrisDill/Raylib-cs#184]
Before submitting a new issue, please verify and check:

 - [x] The issue is specific to Raylib-cs and not raylib
 - [x] I checked there is no similar issue already reported
 - [x] My code has no errors or misuse of Raylib-cs

### Issue description

Since 4.5.0.3, drawing utf8 text shows wrong characters.

### Environment

I'm on Windows, with last version of Raylib-cs.

### Issue screenshot

4.5.0.2 :
![image](https://github.com/ChrisDill/Raylib-cs/assets/16716050/b78697da-5be2-4178-b769-41b9eaaf6c24)

4.5.0.3 :
![image](https://github.com/ChrisDill/Raylib-cs/assets/16716050/02e0c796-1652-4cb2-b3af-79d3b8e82a89)

**Comment by n77y** [comment]
Looks like bug was introduced here #173

**Comment by ChrisDill** [comment]
@AlexisHuvier Can you share a code example for this issue?

**Comment by AlexisHuvier** [comment]
> @AlexisHuvier Can you share a code example for this issue?

I don't have now because it in a big project.
If it's needed, i can make it

**Comment by ChrisDill** [comment]
Just a small example drawing the text that is not showing correctly and the name of the font used.

**Comment by AlexisHuvier** [comment]
> Just a small example drawing the text that is not showing correctly and the name of the font used.

Screens in the first message used extern font but with DrawText there is the same problem. So i used it to make this example :

```csharp
    private static void Main()
    {
        // Initialization
        //--------------------------------------------------------------------------------------
        const int screenWidth = 800;
        const int screenHeight = 450;

        InitWindow(screenWidth, screenHeight, "Test");
        //--------------------------------------------------------------------------------------

        // Main game loop
        while (!WindowShouldClose())
        {
            BeginDrawing();
            ClearBackground(Color.RAYWHITE);

            DrawText("éèàùè&", 250, 20, 20, Color.DARKGRAY);

            EndDrawing();
        }

        CloseWindow();
    }
```

Result with 4.5.0.3 :
![image](https://github.com/ChrisDill/Raylib-cs/assets/16716050/c9f3a267-19db-4172-83d7-d58842833082)

**Comment by n77y** [comment]
Wouldn’t this be expected with the default font? I don’t believe there are any additional characters beyond ASCII in the default font.

**Comment by AlexisHuvier** [comment]
> Wouldn’t this be expected with the default font? I don’t believe there are any additional characters beyond ASCII in the default font.

With the same code, 4.5.0.2 make this result :
![image](https://github.com/ChrisDill/Raylib-cs/assets/16716050/3ec58171-7dd8-4c5c-9436-0f2e3094f4de)

**Comment by n77y** [comment]
I'm proven wrong about the default font.

I have made a pr with a fix for this bug #185.

This was the result of a fix for another bug. That fix changed all the string conversions from UTF8 to ANSI. Issue is that raylib uses UTF8 for drawing text not ANSI.

**Comment by ChrisDill** [comment]
@n77y  Thanks for looking into this issue. Few changes needed on your pr then it can be merged.

**Comment by ChrisDill** [comment]
@AlexisHuvier @n77y Fix is now in the 4.5.0.4 release.
